installer/pkg/config/v1/config.go

```diff
@@ -216,6 +218,10 @@ type WorkspaceTemplates struct {
 	Probe      *corev1.Pod `json:"probe"`
 }
 
+type Meta struct {
+	GitHubApp *ObjectRef `json:"githubApp,omitempty"`
```

Comment:
How are SH users expected to use this? Are there docs that describe how to enable prebuilds?

Reply:
> Thank you for applying this formatting :)

I know how to make you happy.

> How are SH users expected to use this?

There will be, as part of this PR when it's ready for review